requirements-docs.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx
```

Comment:
after trying to test this I realized that this doesn't actually remove anything from `requirements.txt` - I think this should also remove `Sphinx` from `requirements.txt`

Reply:
I can't do that yet, because it will break readthedocs.

Once this is merged I'll update readthedocs to point to the new requirements file and remove the documentation requirements from requirements.txt.